rc/lsp.kak

```diff
@@ -342,9 +342,39 @@ have_kakoune_feature_filtertext = ${kak_opt_lsp_have_kakoune_feature_filtertext}
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }}
 
+declare-option -hidden str lsp_completion_selection ""
+
 define-command -hidden lsp-completion-dismissed -docstring "Called when the completion pager is closed" %{
+    set-option window lsp_completion_selection %val{hook_param}
+    try %{
+        eval -draft %{
```

Comment:
please use long names here ("evaluate-commands").
(The only exception is the snippets code, which was vendored from occivink's snippets plugin.)